elasticsearch/sgconfig/config.yml

```diff
@@ -12,39 +12,12 @@ opendistro_security:
         order: 0
         http_authenticator:
           challenge: false
-          type: io.fabric8.elasticsearch.plugin.auth.OpenShiftTokenAuthentication
+          type: proxy
```

Comment:
Since you are making the change, you might as well go and update the config to what we need https://github.com/opendistro-for-elasticsearch/security/pull/85  The DLS isnt in place so I expect it to function the same as `proxy`

Reply:
`extended_proxy` ?

Reply:
> `extended_proxy` ?

dash https://github.com/opendistro-for-elasticsearch/security/pull/85/files#diff-062271cb239849b80aa8fb7d9b55487cR217